locale/shine/extensions/ban/ruRU.json

```diff
@@ -0,0 +1,21 @@
+{
+	"ADD_BAN": "Добавить блокировку",
+	"SELECT_PLAYER": "Выбрать игрока.",
+	"DURATION_LABEL": "Длительность блокировки (в минутах, 0 бессрочная блокировка):",
+	"DURATION_HINT": "Задайте длительность блокировки...",
+	"DURATION_PERMANENT": "Длительность: Навсегда.",
+	"DURATION_TIME": "Длительность: {Time}",
+	"REASON": "Причина:",
+	"Bans": "Блокировки",
+	"BANNED_BY": "Заблокирован",
+	"EXPIRY": "Истекает",
+	"NEVER": "Никогда",
+	"DATE_FORMAT": "%д %м %г %ч:%м",
+	"UNBAN": "Разблокировать",
+	"LOAD_MORE": "Загрузить ещё",
+	"LOAD_MORE_TIP": "Загружает больше данных с сервера.",
+	"PLAYER_BANNED": "заблокирован {TargetName} {Duration:NonZero:for }{Duration:Duration:permanently} ({Reason}).",
```

Comment:
This message will have a bit of English in the middle. The `{Duration:NonZero:for }` bit means, if `Duration` is not zero, replace with `for `. Similarly for the `{Duration:Duration:permanently}`, this will display `permanently` if duration is 0.

I'm not sure how it should be formatted in Russian, but the idea in English is that it ends up with either:
`banned SomePlayerName for 5 hours` or `banned SomePlayerName permanently`.